generators/generator-multibot/README.md

```diff
@@ -0,0 +1,32 @@
+# generator-conversational-core [![NPM version][npm-image]][npm-url]
+> test generator for conversational core preview bot template
+
+## Installation
+
+First, install [Yeoman](http://yeoman.io) and generator-conversational-core using [npm](https://www.npmjs.com/) (we assume you have pre-installed [node.js](https://nodejs.org/)).
+
+```bash
+npm install -g yo
+npm install -g generator-conversational-core
+```
+
+Then generate your new project:
+
+```bash
+yo conversational-core
+```
+
+## Getting To Know Yeoman
+
+ * Yeoman has a heart of gold.
+ * Yeoman is a person with feelings and opinions, but is very easy to work with.
+ * Yeoman can be too opinionated at times but is easily convinced not to be.
+ * Feel free to [learn more about Yeoman](http://yeoman.io/).
+
+## License
```

Comment:
And all this

Reply:
Actually this whole readme should be simplified (I know the generator spat this out but I have since updated it)

Eyal was confused by these auto generated steps and they are not really valid for our scenarios. I have updated conversational core readme to just have like a simple Composer centric readme